backends/cuda/ceed-cuda-operator.c

```diff
@@ -1204,11 +1204,11 @@ int CeedOperatorCreate_Cuda(CeedOperator op) {
   ierr = CeedSetBackendFunction(ceed, "Operator", op, "LinearAssembleQFunction",
                                 CeedOperatorLinearAssembleQFunction_Cuda);
   CeedChk(ierr);
-  ierr = CeedSetBackendFunction(ceed, "Operator", op, "LinearAssembleDiagonal",
+  ierr = CeedSetBackendFunction(ceed, "Operator", op, "LinearAssembleAddDiagonal",
                                 CeedOperatorLinearAssembleDiagonal_Cuda);
   CeedChk(ierr);
   ierr = CeedSetBackendFunction(ceed, "Operator", op,
-                                "LinearAssemblePointBlockDiagonal",
+                                "LinearAssembleAddPointBlockDiagonal",
                                 CeedOperatorLinearAssemblePointBlockDiagonal_Cuda);
```

Comment:
Should the function names be chosen to match so this doesn't cause confusion in the future?

Reply:
Reasonable. We should really do the same with Apply as well.

Reply:
Updated. It wasn't there before because I was being sloppy when I updated Apply->ApplyAdd and Assemble -> AssembleAdd in the interface.